js/exporter/jspdf/export_data_grid.js

```diff
@@ -0,0 +1,31 @@
+import { isDefined, isObject } from '../../core/utils/type';
+import { Export } from './export';
+import DataGrid from '../../ui/data_grid';
+
+function exportDataGrid(options) {
+    return Export.export(_getFullOptions(options));
+}
+
+function _getFullOptions(options) {
+    if(!(isDefined(options) && isObject(options))) {
+        throw Error('The "exportDataGrid" method requires a configuration object.');
+    }
+    if(!(isDefined(options.component) && isObject(options.component) && options.component instanceof DataGrid)) {
+        throw Error('The "component" field must contain a DataGrid instance.');
+    }
+    if(!isDefined(options.selectedRowsOnly)) {
+        options.selectedRowsOnly = false;
+    }
+    if(!isDefined(options.autoFilterEnabled)) {
```

Comment:
autoFilterEnabled for pdf? Delete?

Reply:
Yes, this option is not needed here.
Removed